File: smswa.accelerator/smsmwa.accelerator.web/Scripts/models/addAddress.js
define(['knockout', 'validation'], function (ko, validation) {

    var model = function() {
        
        this.model = {
            line1: '',
            line2: '',
            suburb: '',
            postcode: '',
            state: '',
            country: ''
        }
        this.boundModel = ko.mapping.fromJS(this.model);
        
        this.validation = new validation();
        this.line1Validator = this.validation
            .createItem(this.boundModel.line1, "Line 1")
            .addRequiredFieldValidator();

    }

    return model;
});

File: smswa.accelerator/smsmwa.accelerator.web/Models/AddAddress.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace smsmwa.accelerator.web.Models
{
    public class AddAddress
    {
        public int Id { get; set; }
        public int AddressTypeId { get; set; }
        public string Line1 { get; set; }
        public string Line2 { get; set; }
        public string Suburb { get; set; }
        public string Postcode { get; set; }
        public string State { get; set; }
        public string Country { get; set; }
    }
}